src/adapt/expr.js

```diff
@@ -24,7 +24,7 @@ goog.require('adapt.base');
 /**
  * @typedef {{fontFamily:string, lineHeight:number, margin:number, hyphenate:boolean,
  *   	columnWidth:number, horizontal:boolean, nightMode:boolean, spreadView:boolean,
- *      pageBorder:number, enabledMediaTypes:!Object.<string,boolean>}}
+ *      pageBorder:number, enabledMediaTypes:!Object.<string,boolean>,defaultPaperSize:(Object<number,number>|undefined)}}
```

Comment:
I think it should be `Object<string,number>` rather than `Object<number,number>`.